SilKit/IntegrationTests/Hourglass/CMakeLists.txt

```diff
@@ -46,6 +51,10 @@ endif ()
 
 add_silkit_test_executable(SilKitHourglassTests)
 
+target_compile_definitions(SilKitHourglassTests
+    PRIVATE SIL_KIT_TEST_MAIN_DO_NOT_PRINT_VERSION
```

Comment:
Is this for performance reasons? many small tests and std I/O overhead?

Reply:
No this is because the hourglass tests implement `SilKit_Version_String` (among all the other C-functions) with a call to a `MOCK_METHOD` (see `MockCapi.hpp`, `MockCapi.cpp`, and `MockCapiTest.hpp`.

This code crashes if the underlying global pointer hasn't been initialized (which is only done when the test starts).